src/braket/device_schema/quera/quera_ahs_paradigm_properties_v1.py

```diff
@@ -0,0 +1,328 @@
+        phaseRange(List[Decimal]): Achievable phase range for the global
+            Rydberg pulse
+        phaseResolution(Decimal): Resolution with which global Rabi frequency
+            phase can be specified
+        phaseSlewRateMax(Decimal): Maximum slew rate for phase
+        timeResolution(Decimal): Resolution with which times for global
+            Rydberg drive parameters can be specified
+        timeDeltaMin(Decimal): Minimum time step with which times for
+            global Rydberg drive parameters can be specified
+    """
+
+    rabiFrequencyRange: List[Decimal]
+    rabiFrequencyResolution: Decimal
+    rabiFrequencySlewRateMax: Decimal
+    detuningRange: List[Decimal]
+    detuningResolution: Decimal
+    detuningSlewRateMax: Decimal
+    phaseRange: List[Decimal]
+    phaseResolution: Decimal
+    phaseSlewRateMax: Decimal
+    timeResolution: Decimal
+    timeDeltaMin: Decimal
+
+
+class RydbergLocal(BaseModel):
+    """
+    Rydberg Local
+    Attributes:
+        detuningRange (List[Decimal]): Achievable detuning range for
+            local detuning patterns
+        commonDetuningResolution(Decimal): Resolution with which time-dependent
+            term of local detuning can be specified
+        localDetuningResolution(Decimal): Resolution with which site-dependent
+            coefficients can be specified
+        detuningSlewRateMax(Decimal): Maximum slew rate for changing the local detuning
+        numberLocalDetuningSites (int): Max number of sites available for the
+            local Rydberg detuning pattern
+        spacingRadialMin (Decimal): Minimum radial spacing between any two
+            sites in the local detuning pattern
+        timeResolution(Decimal): Resolution with which times for local Rydberg
+            drive parameters can be specified
+        timeDeltaMin(Decimal): Minimum time step with which times for local
+            Rydberg drive parameters can be specified
+    """
+
+    detuningRange: List[Decimal]
+    commonDetuningResolution: Decimal
+    localDetuningResolution: Decimal
+    detuningSlewRateMax: Decimal
+    numberLocalDetuningSites: int
+    spacingRadialMin: Decimal
+    timeResolution: Decimal
+    timeDeltaMin: Decimal
+
+
+class Rydberg(BaseModel):
+    """
+    Rydberg Model
+    Attributes:
+        c6Coefficient (Decimal): Rydberg-Rydberg C6 interaction coefficient
+        timeMax (Decimal): Maximum duration of Rydberg drive
+        rydbergGlobal: Rydberg Global
+        rydbergLocal: Rydberg Local
+    """
+
+    c6Coefficient: Decimal
+    timeMax: Decimal
+    rydbergGlobal: RydbergGlobal
+    rydbergLocal: RydbergLocal
+
+
+class PerformanceRydbergGeometry(BaseModel):
+    """
+    Performance of Rydberg Geometry
+    Attributes:
+        positionErrorAbs (Decimal): Error between target and actual site position
+    """
+
+    positionErrorAbs: Decimal
+
+
+class PerformanceRydbergGlobal(BaseModel):
+    """
+    Performance of Rydberg Global
```

Comment:
A better description would be: "Parameters determining the limitations of the global driving field"